heroku_connect/management/commands/load_remote_schema.py

```diff
@@ -0,0 +1,87 @@
+import os
+import re
+import subprocess
+
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    """
+    Load schema from remote Heroku PostgreSQL database.
+
+    This command can be useful to load the Heroku Connect database schema into
+    a local development environment.
+
+    Example::
+
+        python manage.py load_remote_schema --app ninja | psql -a
+
+    .. note::
+
+        This command requires the `Heroku CLI`_ and PostgreSQL_ to be installed.
+
+    .. _`Heroku CLI`: https://cli.heroku.com/
+    .. _PostgreSQL: https://www.postgresql.org/
+
+    """
+
+    help = __doc__.strip().splitlines()[0]
+
+    url_pattern = r'postgres://(?P<user>[\d\w]*):(?P<passwd>[\d\w]*)' \
+                  r'@(?P<host>[^:]+):(?P<port>\d+)/(?P<dbname>[\d\w]+)'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--app', '-a', dest='HEROKU_APP', type=str,
+                            help="Heroku app name that the schema will be pulled from.")
+        parser.add_argument('--schema', '-s', dest='SCHEMA_NAME', type=str, default='salesforce',
+                            help="Name of schema that you want to load.")
+
+    def handle(self, *args, **options):
+        heroku_app = options.get('HEROKU_APP')
+        schema_name = options['SCHEMA_NAME']
+        url = self.get_database_url(heroku_app)
+        credentials = self.parse_credentials(url)
+        schema = self.get_schema(**credentials, schema_name=schema_name)
+        self.stdout.write(schema)
+
+    @staticmethod
+    def get_database_url(heroku_app):
+        run_args = ['heroku', 'pg:credentials:url']
+        if heroku_app:
```

Comment:
If you have only a single remote setup for you repository that has this package installed, this will be the default. It doesn't work for us, because we have multiple environments setup usually. But I can work for some.

Reply:
:+1: